dotcom-rendering/package.json

```diff
@@ -96,16 +96,18 @@
     "@types/lodash.debounce": "^4.0.6",
     "@types/lodash.escape": "^4.0.6",
     "@types/lodash.get": "^4.4.4",
-    "@types/node": "^14.14.24",
-    "@types/node-fetch": "^2.6.0",
+    "@types/node": "^14.18.3",
+    "@types/node-fetch": "^2.6.1",
     "@types/raven-js": "^3.10.0",
     "@types/react": "17.0.3",
     "@types/react-dom": "17.0.3",
     "@types/react-test-renderer": "17.0.1",
     "@types/response-time": "^2.3.4",
     "@types/sanitize-html": "^1.27.1",
-    "@types/webpack": "^4.41.26",
+    "@types/uuid": "^8.3.4",
+    "@types/webpack-bundle-analyzer": "^4.4.1",
     "@types/webpack-env": "^1.16.0",
+    "@types/webpack-node-externals": "^2.5.3",
```

Comment:
DCR uses `"webpack-node-externals": "^3.0.0"`, is this the correct types version for that?

Reply:
This actually breaks the DefinitelyTyped contract… but they have been updated since then! https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/webpack-node-externals